app/main.py

```diff
@@ -1,13 +1,30 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    # """ formats a single error dictionary """
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8",
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    # """ formats all errors for a particular file
+    # and adds the path key — path to the file,
+    # and the status key — "failed" if there are errors,
+    # "passed" if there are no errors:
+    # """
```

Comment:
if you are using docstrings it's not needed to add them as comments using #
```suggestion
    """ formats all errors for a particular file
    and adds the path key — path to the file,
    and the status key — "failed" if there are errors,
    "passed" if there are no errors:
    """
```

Reply:
https://www.programiz.com/python-programming/docstrings

Reply:
i know it ,
but  normal docstring dont  permit your  test ,
and i sended report about  it  already.

So , i must delete docstrings at all ?